assets/js/src/integration/formidableforms.js

```diff
@@ -0,0 +1,24 @@
+/***********************************
+ * Copyright (c) 2020, Popup Maker
+ **********************************/
+
+{
+	const formProvider = "formidableforms";
+	const $ = window.jQuery;
+
+	$(document).ready(() => {
```

Comment:
Should you go ahead and write this for jQuery v3 to save us some time later? @fpcorso

Reply:
Also for that matter, your using event delegation via `$(document).on`. There is no need to put it inside `$(document).ready()`. It listens for all events and if the event handler name matches it runs the callback. Its a nuance part of how JS events work with bubbling because all events start on the element clicked and bubble up to the document, then back down again to the element.